node/lib/packaging/PackageInfo.ts

```diff
@@ -0,0 +1,173 @@
+
+/**
+ * Information about an OpenT2T package, as loaded from the package.json.
+ */
+export class PackageInfo {
```

Comment:
Should we have 'isExperiental' property (or similar.. like in WinRT ) to declare a package that is still being prototyped. ?